### ISSUE TITLE
About test precision.

### ISSUE THREAD

**Issue by chengfeng0113** [WIKI2020/FacePose_pytorch#8]
Using 2D key points to solve the problem of face pose is not a good solution, especially in the case of large pose, such as yaw greater than 60 degrees. After testing, it is found that sometimes the face detector provided by you can not detect the face with large angle, and as long as the yaw is greater than 45 degrees, there will be an error, so we look forward to your higher precision version.In addition, the stability of pose is not very good.

**Comment by WIKI2020** [comment]
Yes, when it is greater than 65 degrees, it will start to be undetectable or have errors, which is the problem of all key point detection algorithms.However, in the actual commercial product, it is meaningless to detect the turning direction degree greater than 65; the new model will be very accurate in the range of 0 ~ 65 degrees, and will be updated in October or November

**Comment by chengfeng0113** [comment]
Maybe the pain point of solving the problem is different. I need to pay attention to the extremely large angle face. In addition, do you label the 2D face key point data by yourself? I have a question. When yaw is greater than 45 degrees, the key points on the cheek are invisible. If only visible key points are used, the calculated pose angle is inaccurate. How do you solve this problem? Can you tell me if you can?

**Comment by WIKI2020** [comment]
(1) Key point data is labeled by ourselves
(2) The new model will be very accurate （about -2°~+2°）in the range of 0 ~ 65 degreecan，and also can predict large angle, but the error（about -6°~+6°） will occur when the angle is greater than 65 degrees

**Comment by chengfeng0113** [comment]
I see,thank you!

**Comment by ZengZhaoxiang** [comment]
Did you test on some public datasets such as BIWI and ALFW2000？

**Comment by WIKI2020** [comment]
@ZengZhaoxiang No, the existing open source datasets don't have accurate facial marker angles; the data are all made  and tested by ourselves
